docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -1717,8 +1749,890 @@ Here are the steps:
+
+   Observations:
+     - We are leaving the namespace `sample-domain1-ns` unchanged for the ConfigMap because we plan to deploy domain `sample-domain2` to the same namespace as `sample-domain1`.
+     - We name and label the ConfigMap using its associated domain UID for two reasons:
+       - To make it obvious which ConfigMap belongs to which domain.
+       - To make it easier to cleanup a domain. Typical cleanup scripts use the `weblogic.domainUID` label as a convenience for finding all resources associated with a domain.
+     - We use a different ConfigMap for the new domain for two reasons:
+       - To make it easier to keep the life-cycle and/or CI/CD process for the two domains simple and independent. Any shared resources between the two domains must necessarily have their own life-cycle.
+       - To 'future proof' the new domain so that changes to the original domain or new domain's ConfigMap can be independent.
+
+
+1. Create the secrets that are referenced by the WDT model files in the image and ConfigMap, and that will be referenced by the domain resource.
+
+   Run the following commands:
+   ```
+   $ # spec.webLogicCredentialsSecret
+   $ kubectl -n sample-domain1-ns create secret generic \
+     sample-domain2-weblogic-credentials \
+      --from-literal=username=weblogic --from-literal=password=welcome1
+   $ kubectl -n sample-domain1-ns label  secret \
+     sample-domain2-weblogic-credentials \
+     weblogic.domainUID=sample-domain2
+
+   $ # spec.configuration.model.runtimeEncryptionSecret
+   $ kubectl -n sample-domain1-ns create secret generic \
+     sample-domain2-runtime-encryption-secret \
+      --from-literal=password=my_runtime_password
+   $ kubectl -n sample-domain1-ns label  secret \
+     sample-domain2-runtime-encryption-secret \
+     weblogic.domainUID=sample-domain2
+
+   $ # referenced by spec.configuration.secrets and by the datasource model YAML in the ConfigMap
+   $ kubectl -n sample-domain1-ns create secret generic \
+     sample-domain2-datasource-secret \
+      --from-literal=password=Oradoc_db1 \
+      --from-literal=url=jdbc:oracle:thin:@oracle-db.default.svc.cluster.local:1521/devpdb.k8s
+   $ kubectl -n sample-domain1-ns label  secret \
+     sample-domain2-datasource-secret \
+     weblogic.domainUID=sample-domain2
+   ```
+
+   Observations:
+     - We are leaving the namespace `sample-domain1-ns` unchanged for each secret because we plan to deploy domain `sample-domain2` to the same namespace as `sample-domain1`.
+     - We name and label the secrets using their associated domain UID for two reasons:
+       - To make it obvious which secret belongs to which domain.
+       - To make it easier to cleanup a domain. Typical cleanup scripts use the `weblogic.domainUID` label as a convenience for finding all resources associated with a domain.
```

Comment:
to cleanup a domain -> to clean up a domain (clean up (v), cleanup (adj) (n))

Reply:
done